katas/content/multi_qubit_measurements/index.md

```diff
@@ -9,18 +9,22 @@ In the previous kata, we discussed the concept of measurements done on single-qu
 Building upon those ideas, this kata will introduce you to measurements done on multi-qubit systems, and how to implement such measurements in Q#.
 This will include measuring a single qubit in a multi-qubit system, as well as measuring multiple qubits simultaneously.
 
-We recommend to go through the kata that introduces single-qubit system measurements before starting this one.
-$\renewcommand{\ket}[1]{\left\lvert#1\right\rangle}$
-$\renewcommand{\bra}[1]{\left\langle#1\right\rvert}$
+**This tutorial covers the following topics:**
```

Comment:
nit: Matt has mentioned that we should use the term "kata" instead of "tutorial" throughout the content. This is not something that has to be done in this PR though. It's already captured in the feedback for each kata so we can address it separately.

Reply:
Let's do it as a separate PR.